dev/com.ibm.ws.org.apache.cxf.cxf.api.2.6.2/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,280 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.methodParameters=do not generate
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
```

Comment:
Same as above, since we want to make sure it retains the same 1.7 compatibility?